### PR TITLE
Copy text

### LINKED ISSUE #38: Copy component
Required for 
- TXs new
- Allowances

### PR DESCRIPTION
**Pull request by nicosampler**
Closes #38.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr31--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr31--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr31--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr31--safereactcomponents.review.gnosisdev.com

**Comment by nicosampler** [comment]
@mikheevm ready for review

File: src/utils/CopyToClipboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  cursor: pointer;
+  outline: inherit;
+`;
+
+type Props = {
+  textToCopy: string;
+  className?: string;
+};
+
+const CopyToClipboard = ({
```
Comment (mikheevm) [inline src/utils/CopyToClipboard/index.tsx:22]: `CopyBtn`? feels weird to have `copyToClipboard` and `CopyToClipboard` in the same file

File: src/utils/CopyToClipboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
```
Comment (mikheevm) [inline src/utils/CopyToClipboard/index.tsx:1]: Why is this placed in utils? Shouldn't it be in inputs like all other buttons are?

Comment (nicosampler) [reply src/utils/CopyToClipboard/index.tsx:1]: I see it more like and utility than input, but I don't have strong opinions about it

File: src/utils/CopyToClipboard/index.tsx (hunk 1 above)
Comment (nicosampler) [reply src/utils/CopyToClipboard/index.tsx:22]: I think CopyToClipboard is much explicit than just CopyBtn. I renamed the function instead.

File: src/utils/CopyToClipboard/index.tsx (hunk 2 above)
Comment (nicosampler) [reply src/utils/CopyToClipboard/index.tsx:1]: Although all the components inside the input category should be used inside a form. This is not the case.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr31--safereactcomponents.review.gnosisdev.com

File: src/utils/CopyToClipboard/index.tsx (hunk 2 above)
Comment (mikheevm) [reply src/utils/CopyToClipboard/index.tsx:1]: So we shouldn’t use the button from `inputs` for triggering scripts? Can we point this in a documentation?

Comment (mikheevm) [reply src/utils/CopyToClipboard/index.tsx:1]: I think the structure of the components should be refactored, could use evergreen as an example: https://evergreen.segment.com/components

Comment (nicosampler) [reply src/utils/CopyToClipboard/index.tsx:1]: > So we shouldn’t use the button from inputs for triggering scripts? Can we point this in a documentation?

I think we should, what I want to say is that input category is more related (not exclusive) to inputs form. As I user of the lib, if I want to search for a copy button, I would search first in utilities before inputs. But again, just my opinion.

Comment (nicosampler) [reply src/utils/CopyToClipboard/index.tsx:1]: > I think the structure of the components should be refactored, could use evergreen as an example: https://evergreen.segment.com/components

We discussed this before and we decided to follow the material-ui structure. but sure, we can change that, please create an issue and we can reorganize the categories.

File: src/utils/CopyToClipboard/index.tsx (hunk 1 above)
Comment (mikheevm) [reply src/utils/CopyToClipboard/index.tsx:22]: Explicit for what? What is CopyToClipboard component? A button? A text which would be copied if you click on it?

Comment (nicosampler) [reply src/utils/CopyToClipboard/index.tsx]: @mikheevm renamed.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr31--safereactcomponents.review.gnosisdev.com

**Pull request merged**